composer/utils/sftp_object_store.py

```diff
@@ -0,0 +1,104 @@
+# Copyright 2022 MosaicML Composer authors
+# SPDX-License-Identifier: Apache-2.0
+
+"""Utility for uploading to and downloading from cloud object stores."""
+import io
+import os
+import sys
+import uuid
+import warnings
+from typing import Iterator, Optional, Union
+
+from paramiko import SFTPAttributes, SSHClient
+
+from composer.utils.iter_helpers import IteratorFileStream
+
+__all__ = ["SFTPObjectStore"]
+
+
+class SFTPObjectStore:
+    """Utility for uploading to and downloading to a server via SFTP."""
+
+    def __init__(self, host: str, port: int = 22, username: Optional[str] = None, key_file_path: Optional[str] = None):
+        self.ssh_client = SSHClient()
+
+        # add AutoAddPolicy
+        if key_file_path is None:
```

Comment:
Do we need to do this check? Or can we do `ssh_client.load_system_host_keys(host_key_filename)`?

Reply:
Fixed